transmogrifier/sources/ead.py

```diff
@@ -252,6 +252,48 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
                         )
                     )
 
+        # rights
+        for rights_element in collection_description.find_all(
+            ["accessrestrict", "userestrict"], recursive=False
+        ):
+            if rights_value := self.create_string_from_mixed_value(
+                rights_element, " ", ["head"]
+            ):
+                fields.setdefault("rights", []).append(
+                    timdex.Rights(
+                        description=rights_value,
+                        kind=self.crosswalk_type_value(rights_element.name),
+                    )
+                )
+
+        # subjects
+        for control_access_element in control_access_elements:
+            for subject_element in control_access_element.find_all(
```

Comment:
Do we want to exclude `genreform` and `geogname` from subjects since we've extracted those into other fields? I think it could go either way and I don't have strong feelings about it...